megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -937,19 +964,19 @@ private synchronized void manageMenu() {
             viewZoomOut.setEnabled(false);
             viewKeybindsOverlay.setEnabled(false);
         }
+        
+        fileUnitsPaste.setEnabled(phase == IGame.Phase.PHASE_LOUNGE);
 
         // If we have a unit list, and if we are in the lounge,
         // then we can still perform all unit list actions.
         if (hasUnitList) {
             fileUnitsOpen.setEnabled(phase == IGame.Phase.PHASE_LOUNGE);
-            fileUnitsClear.setEnabled(phase == IGame.Phase.PHASE_LOUNGE);
           //  fileUnitsSave.setEnabled(phase == IGame.Phase.PHASE_LOUNGE);
```

Comment:
Maybe get rid of the commented out code here and on line 980? Or address it somehow otherwise.

Reply:
I think I'll remove these menu items. They are for the lobby only anyway and saving unit lists depends on blind drop options which the menu bar doesnt adress and loading units lists depends on the selected player which the menu bar also doesnt adress. There's big buttons for loading/saving.